etc/docker/Dockerfile.triton-jetpack-focal

```diff
@@ -135,15 +136,19 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Build and install the Viam Triton MLModelService from our
 # local state against the C++ SDK and Triton Server
 ADD . ${HOME}/opt/src/viam-mlmodelservice-triton
+WORKDIR ${HOME}/opt/src/viam-mlmodelservice-triton
 RUN \
-    cd ${HOME}/opt/src/viam-mlmodelservice-triton && \
     cmake -S . -B build -G Ninja \
-      -DVIAM_MLMODELSERVICE_TRITON_TRITONSERVER_ROOT=/opt/tritonserver/tritonserver \
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=on \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DCMAKE_PREFIX_PATH=/opt/viam && \
     cmake --build build --target install -- -v && \
     cmake --install build --prefix /opt/viam
 
+FROM build AS linter
+RUN apt-get install -y clang-tidy
+WORKDIR ${HOME}/opt/src/viam-mlmodelservice-triton/src
+RUN clang-tidy -p ../build *.cpp *.hpp
```

Comment:
Did you investigate https://cmake.org/cmake/help/latest/prop_tgt/LANG_CLANG_TIDY.html#prop_tgt:%3CLANG%3E_CLANG_TIDY to have CMake drive linting?

Reply:
No, once I got this line to work, I didn't bother trying to make it more complicated.